### PR TITLE
Rename 'foo-procedures' to 'banana-procedures'

### PR DESCRIPTION
**Pull request by Mythicaeda**
* **Tickets addressed:** None
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

These procedures use the Bananation mission model, not the Foo mission model. The old module name was confusing.

## Verification

I was able to build the module using the command in the README. I also did a project-wide search and found no other instances of either `foo-procedures` or `fooprocedures`

## Documentation

I saw no mention of this module in the Aerie Docs website and have updated the README to use the new name.

### CODE REVIEW

File: procedural/examples/banana-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/bananaprocedures/constraints/Hello.java
Comment (JoelCourtney) [inline procedural/examples/banana-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/bananaprocedures/constraints/Hello.java:1]: I think this is pretty much useless at this point. We used it as an early test of jar building before we had actual goal classes. Can you delete it please

**Review by JoelCourtney — APPROVED** [review]
I have a request, but I'm approving anyway because its not actually on the merits of the PR

**Review by mattdailis — APPROVED** [review]
Good call

File: procedural/examples/banana-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/bananaprocedures/constraints/Hello.java
Comment (Mythicaeda) [reply procedural/examples/banana-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/bananaprocedures/constraints/Hello.java:1]: Done!

**Pull request merged**